meson.build

```diff
@@ -0,0 +1,491 @@
+project('vips', 'c', 'cpp',
+    version: '8.13.0',
+    meson_version: '>=0.56',
+    default_options: [
+        'c_std=c99'
+    ]
+)
+
+version_parts = meson.project_version().split('.')
+version_major = version_parts[0]
+version_minor = version_parts[1]
+version_patch = version_parts[2]
+
+# rules:
+# sources changed: increment revision
+# binary interface changed: increment current, reset revision to 0
+#   binary interface changes backwards compatible?: increment age
+#   binary interface changes not backwards compatible?: reset age to 0
+library_current = 57
+library_age = 15
+library_revision = 0
+library_version = '@0@.@1@.@2@'.format(library_current - library_age, library_age, library_revision)
+
+gnome = import('gnome')
+pymod = import('python')
+pkg = import('pkgconfig')
+i18n = import('i18n')
+
+add_project_arguments('-Drestrict=__restrict', language: 'cpp')
+
+cc = meson.get_compiler('c')
+
+glib_dep = dependency('glib-2.0')
+gio_dep = dependency('gio-2.0')
+gobject_dep = dependency('gobject-2.0')
+gmodule_dep = dependency('gmodule-no-export-2.0')
+expat_dep = dependency('expat')
+m_dep = cc.find_library('m', required : false)
+
+libvips_deps = [
+    glib_dep,
+    gio_dep,
+    gobject_dep,
+    gmodule_dep,
+    expat_dep,
+    m_dep,
+]
+
+prefix_dir = get_option('prefix')
+lib_dir = prefix_dir / get_option('libdir')
+modules_enabled = get_option('modules').require(
```

Comment:
I see an error here with meson 0.56.2 on ubuntu 21.10:

```
meson.build:51:0: ERROR: Unknown method "require" in object.
```

Is this a later feature?

Reply:
Ah indeed, this is a 0.59 feature, I've reworked it to be able to use 0.56, thanks for trying it